.github/workflows/compound-python.yml

```diff
@@ -15,11 +15,11 @@ on:
       pythonVersion:
         required: false
         type: string
-        default: '3.8'
+        default: '3.11'
       python-check-versions:
         required: false
         type: string
-        default: "['3.8', '3.9', '3.10', '3.11', '3.12']"
```

Comment:
question: why do we remove python 3.12? I think we should keep it as it us the latest stable release